### PR TITLE
JBDS 9.1.0.GA release blog

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

**Comment by alexeykazakov** [comment]
@nickboldt @maxandersen please review

**Comment by nickboldt** [comment]
Skimmed, looks good. No obvious typos. 

Should the title be "JBoss Tools and Red Hat Developer Studio for Eclipse Mars" or "... for Mars.2" ?

AERI copy is awesome. Gold star to the author of that section. :)

File: blog/ga-for-mars2.adoc [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+== Installation
+
+JBoss Developer Studio comes with everything pre-bundled in its installer. Simply download it from our https://www.jboss.org/products/devstudio.html[JBoss Products page] and run it like this:
+ 
+    java -jar jboss-devstudio-<installername>.jar
+
+JBoss Tools or Bring-Your-Own-Eclipse (BYOE) JBoss Developer Studio require a bit more:
+
+This release requires at least Eclipse 4.5.2 (Mars.2) but we recommend
+using the http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/marsr[Eclipse 4.5.2 Mars JEE Bundle] since then you get most of the dependencies preinstalled.
```
Comment (fbricon) [inline blog/ga-for-mars2.adoc]: s/JEE/Java EE/g

File: blog/ga-for-mars2.adoc [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+This release is based on Eclipse Mars.2 and our focus was mainly on OpenShift and Red Hat Container Development Kit tools improvements. Besides that there is a bunch of new features and a lot of fixed bugs. 
+The full list of what is new you can find on link:/documentation/whatsnew/jbosstools/4.3.1.Final.html[this page].
+
+=== New Developer Studio Platform Installer for Container Based Development
+
+A new installer for 64-bit Windows is now available. This installer is quite different from the old good Developer Studio universal installer.
+This will help you install Developer Studio with a full development environment for container based development using *Red Hat Enterprise Linux* and *OpenShift*.
+
+The Developer Studio Platform Installer Beta currently works on 64-bit Windows only and is available as a Technology Preview.
```
Comment (fbricon) [inline blog/ga-for-mars2.adoc]: I say remove the leading 'The'

File: blog/ga-for-mars2.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+There is also a new installer available for 64-bit Windows. See details in link:#new-developer-studio-platform-installer-for-container-based-development[New Developer Studio Platform Installer for Container Based Development] section.
+
+== What is new?
+
+This release is based on Eclipse Mars.2 and our focus was mainly on OpenShift and Red Hat Container Development Kit tools improvements. Besides that there is a bunch of new features and a lot of fixed bugs. 
+The full list of what is new you can find on link:/documentation/whatsnew/jbosstools/4.3.1.Final.html[this page].
+
+=== New Developer Studio Platform Installer for Container Based Development
+
+A new installer for 64-bit Windows is now available. This installer is quite different from the old good Developer Studio universal installer.
```
Comment (fbricon) [inline blog/ga-for-mars2.adoc]: s/old good/good old

**Comment by fbricon** [comment]
Except for a couple errors, looks pretty good to me. I love how the OpenShift section was made shorter and straight to the point :-)

**Comment by alexeykazakov** [comment]
Thanks guys! I've fixed all the errors.

**Comment by alexeykazakov** [comment]
Rebased and pushed to master and production.

**Pull request closed without merging**